apps/projects/app/components/Card/Issue.js

```diff
@@ -156,37 +182,59 @@ Issue.propTypes = {
 const StyledIssue = styled.div`
   flex: 1;
   width: 100%;
-  min-width: 600px;
+  /*min-width: 600px;*/
   background: ${theme.contentBackground};
   display: flex;
-  padding-left: 10px;
-  height: 112px;
+  /*padding-left: 10px;*/
+  /*height: 112px;*/
+  height: auto;
   align-items: center;
   border-radius: 3px;
   border: 1px solid ${theme.contentBorder};
   margin-bottom: -1px;
   position: relative;
-  > :nth-child(2) {
-    /* checkbox */
-    margin-right: 21.5px;
-    justify-content: center;
-    z-index: 2;
-  }
 `
-const IssueDetails = styled.div`
+const IssueTitleDetailsBalance = styled.div`
   display: flex;
+  justify-content: space-between;
+  align-items: center;
 `
-const IssueDesc = styled.div`
+const IssueTitleDetails = styled.div`
+  display: flex;
+  flex-direction: column;
+  > :not(:last-child) {
+    margin-bottom: 6px;
+  }
+`
+const IssueData = styled.div`
   display: flex;
   flex: 1 1 auto;
   flex-direction: column;
-  height: 90px;
+  /*height: 90px;*/
   justify-content: space-around;
-  padding: 10px;
+  padding: 12px 12px 12px 0;
+  position: relative;
+`
+const Balance = styled.div`
+  margin-left: 10px;
 `
-const BalanceAndContext = styled.div`
-  margin-right: 20px;
-  display: inline-flex;
+const IssueTitle = styled(Text.Block).attrs({
+  size: 'large',
+})`
+  display: block;
+  display: -webkit-box;
```

Comment:
Why set display twice?

Reply:
Most probably to have the webkit directives below working - to be confirmed whether it is actually necessary.